mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
+    """
+    Given a registered model, this helper will re-log the model along with the wheels of all the
+    the model dependencies (stored along with the model artifacts).
```

Comment:
This function accepts arbitrary model URIs, not just registered model URIs. I.e. it can be used to add library dependencies to a model in MLflow Tracking.

The current method description doesn't really make it clear what this method does precisely. Can we describe its behavior precisely and then explain the use cases it satisfies?